plugins/org.jkiss.dbeaver.ext.sqlite/src/org/jkiss/dbeaver/ext/sqlite/model/SQLiteDataSource.java

```diff
@@ -40,12 +41,24 @@
 
 public class SQLiteDataSource extends GenericDataSource {
 
-    public SQLiteDataSource(DBRProgressMonitor monitor, DBPDataSourceContainer container, GenericMetaModel metaModel)
-        throws DBException
-    {
+    public SQLiteDataSource(
+        @NotNull DBRProgressMonitor monitor,
+        @NotNull DBPDataSourceContainer container,
+        GenericMetaModel metaModel
+    ) throws DBException {
         super(monitor, container, metaModel, new SQLiteSQLDialect());
     }
 
+    public SQLiteDataSource(
+        @NotNull DBRProgressMonitor monitor,
+        @NotNull DBPDataSourceContainer container,
+        @NotNull GenericMetaModel metaModel,
+        @NotNull SQLDialect dialect
+    ) throws DBException {
+        super(monitor, container, metaModel, dialect);
+    }
```

Comment:
Do you need it?

Reply:
This one is needed. See it's extension.

Reply:
Which extension are you talking about? Please, point me out.

Reply:
EE part